Remotion/Web/Development.WebTesting.IntegrationTests/BrowserSessionTest.cs

```diff
@@ -34,18 +34,18 @@ public void DeleteAllCookies_DeletesAllCookies ()
     {
       var browserSession = Start();
 
-      var allCookiesBeforeCreation = ((RemoteWebDriver)browserSession.Context.Browser.Driver.Native).Manage().Cookies.AllCookies;
+      var allCookiesBeforeCreation = ((OpenQA.Selenium.WebDriver)browserSession.Context.Browser.Driver.Native).Manage().Cookies.AllCookies;
```

Comment:
Because Selenium updated it's type names, we no have a nameclash with our namespace "WebDriver", which means the "Selenium.WebDriver" type must be fully qualified. 

I think changing the namespace in re-motion would be a significant break, given the contents. So, @a-ctor  do you see any significant usage for Selenium.WebDriver" in downstream projects that would require additional consideration.

Please document this conflict as a breaking change ticket.

Reply:
I dont' think that this is a big problem. This only happens if you access the "native" Selenium APIs which should be a rare occurrence. And from the consumer perspective this problem only occurs if you import our `WebDriver` namespace. That should be an even rarer occurrence given that we have mostly browser specific classes in that namespace.

Reply:
Okay, so we just need a breaking change documentation. Is OpenQA.Selenium.WebDriver a new type or was it simply never imported in the past?

Reply:
It did not exist before but the `RemoteWebDriver` no longer exists so its more of a rename

Reply:
okay, brekaing change that the new api clashes with our namespace and we recommend using the interface whenever possible.

Reply:
probably also add a breaking change that RemoteWebDriver has been replaced with WebDriver. These may be two separate breaking change tickets.